csrc/quantization/compressed_tensors/int8_quant_kernels.cu

```diff
@@ -31,6 +31,43 @@ static inline __device__ int8_t float_to_int8_rn(float x) {
 #endif
 }
 
+static inline __device__ int32_t float_to_int32_rn(float x) {
+#ifdef USE_ROCM
+  static const auto i32_min =
+      static_cast<float>(std::numeric_limits<int32_t>::min());
+  static const auto i32_max =
+      static_cast<float>(std::numeric_limits<int32_t>::max());
+  // round
+  float dst = std::nearbyint(x);
```

Comment:
`std::nearbyint` respects the current rounding mode, so it would be best if we explicitly set it to use round to nearest, to match the CUDA path exactly

Reply:
Removed, HIP only supports tonearest. Added a comment